library/std/src/net/tcp.rs

```diff
@@ -9,6 +9,9 @@
         target_os = "trusty",
     ))
 ))]
+// Ferrocene addition: disabled temporarily as these fail when running the test suite with coverage
+// instrumentation enabled.
+#[cfg(not(test))]
 mod tests;
```

Comment:
this is fine short-term. long term i would like the following:
- `x run generate-coverage` (or any similar name) so everyone gets a "canonical" report which is what we put in the certification docs. that runs the equivalent of `x test --coverage=library --no-doc library/core` today (possibly with doctests and std after this PR), and omits things that are known to be slow or broken with coverage instrumentation.
- `x test --coverage [... arbitrary args]` which lets you expand coverage reports by adding more paths to the x.py invocation.

i think the current situation is in some sense the worst of both worlds: you have to know exactly how to invoke x.py to get the right report, but we also have hard-coded stuff like this that doesn't allow people to run with additional tests.

Reply:
I'm worried about forgetting this or missing a bug. :( Can we only ignore during coverage somehow?